src/components/displaySettings/displaySettings.template.html

```diff
@@ -7,65 +7,74 @@ <h2 class="sectionTitle">
-            <option value="is">Icelandic</option>
-            <option value="id">Indonesian</option>
-            <option value="it">Italian</option>
-            <option value="ja">Japanese</option>
-            <option value="kk">Kazakh</option>
-            <option value="ko">Korean</option>
-            <option value="lt-LT">Lithuanian</option>
-            <option value="ms">Malay</option>
-            <option value="mr">Marathi</option>
-            <option value="nb">Norwegian Bokmål</option>
-            <option value="fa">Persian</option>
-            <option value="pr">Pirate</option>
-            <option value="pl">Polish</option>
-            <option value="pt">Portuguese</option>
-            <option value="pt-BR">Portuguese (Brazil)</option>
-            <option value="pt-PT">Portuguese (Portugal)</option>
-            <option value="ro">Romanian</option>
-            <option value="ru">Russian</option>
-            <option value="sk">Slovak</option>
-            <option value="sl-SI">Slovenian (Slovenia)</option>
-            <option value="es">Spanish</option>
-            <option value="es_AR">Spanish (Argentina)</option>
-            <option value="es_DO">Spanish (Dominican Republic)</option>
-            <option value="es-419">Spanish (Latin America)</option>
-            <option value="es-MX">Spanish (Mexico)</option>
-            <option value="sv">Swedish</option>
-            <option value="ta">Tamil</option>
-            <option value="th">Thai</option>
-            <option value="tr">Turkish</option>
-            <option value="uk">Ukrainian</option>
-            <option value="ur_PK">Urdu (Pakistan)</option>
-            <option value="vi">Vietnamese</option>
+            <option value="fr">Français</option>
+            <option value="fr-CA">Français (Canada)</option>
+            <option value="gl">Galego</option>
+            <option value="gsw">Schwiizerdütsch</option>
+            <option value="he">עִבְרִית</option>
+            <option value="hi-IN">हिन्दी</option>
+            <option value="hr">Hrvatski </option>
+            <option value="hu">Magyar</option>
+            <option value="id">Bahasa Indonesia</option>
+            <option value="is-IS">Íslenska</option>
+            <option value="it">Italiano</option>
+            <option value="ja">日本語</option>
+            <option value="kk">Qazaqşa</option>
+            <option value="ko">한국어</option>
+            <option value="lt-LT">Lietuvių</option>
+            <option value="lv">Latviešu</option>
+            <option value="mk">Македонски</option>
+            <option value="ml">മലയാളം</option>
+            <option value="mr">मराठी</option>
+            <option value="ms">Bahasa Melayu</option>
+            <option value="nb">Norsk bokmål</option>
+            <option value="ne">नेपाली</option>			
+            <option value="nl">Nederlands</option>
+            <option value="nn">Norsk nynorsk</option>
```

Comment:
Aren't `bokmål` and `nynorsk` titles (need the upper case)?

Reply:
In Wikipedia interwiki both bokmål and nynorsk are lower case. See for example https://no.wikipedia.org/wiki/Norsk (left column, languges list - På andre språk - Norsk nynorsk).